site2/docs/reference-metrics.md

```diff
@@ -435,6 +435,15 @@ All the subscription metrics are labelled with the following labels:
 | pulsar_subscription_msg_rate_out | Gauge | The total message dispatch rate for a subscription (message per second). |
 | pulsar_subscription_msg_throughput_out | Gauge | The total message dispatch throughput for a subscription (byte per second). |
 | pulsar_subscription_msg_ack_rate | Gauge | The total message acknowledgment rate for a subscription (message per second). |
+| pulsar_subscription_last_expire_timestamp | Gauge | The subscription last message expire execution timestamp. |
```

Comment:
```suggestion
| pulsar_subscription_last_expire_timestamp | Gauge | The expiration timestamp of the subscription's last message. |
```
Is this what you meant?

Reply:
That's copy/pasted from javadoc. I guess your proposal is better.